test/evp_test.c

```diff
@@ -182,39 +168,42 @@ static int test_bin(const char *value, unsigned char **buf, size_t *buflen)
 static int test_uint64(const char *value, uint64_t *pr)
 {
     const char *p = value;
-    if (!*p) {
-        fprintf(stderr, "Invalid empty integer value\n");
+
+    if (!TEST_true(*p)) {
+        TEST_info("Invalid empty integer value");
         return -1;
     }
     *pr = 0;
     while (*p) {
-        if (*pr > UINT64_MAX/10) {
-            fprintf(stderr, "Integer string overflow value=%s\n", value);
+        if (*pr > UINT64_MAX / 10) {
+            TEST_error("Integer string overflow value=%s", value);
             return -1;
         }
         *pr *= 10;
-        if (*p < '0' || *p > '9') {
-            fprintf(stderr, "Invalid integer string value=%s\n", value);
+        if (!TEST_true(isdigit(*p)))
             return -1;
-        }
         *pr += *p - '0';
         p++;
     }
     return 1;
 }
 #endif
 
+typedef struct evp_test_method_st evp_test_method;
+typedef struct evp_test_st evp_test;
+typedef struct key_list_st key_list;
```

Comment:
Hmmm...  should we follow that for non-public ones as well?  I'm thinking that our public functions usually have an uppercase prefix while non-public ones usually have a lowercase prefix.  Could the same line of thought be applied here, and the code style amended?